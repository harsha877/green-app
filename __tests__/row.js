import React from 'react';
import { render, fireEvent, waitFor } from '@testing-library/react-native';
import Row from '../src/components/row'

test("Row Component Loading",()=>{
    render(<Row row={{quizName:"waterQuiz",customerName:"mike"}} onPressHandler={()=>{}}/>)
 
 })
 
 test("Row component snapshot",()=>{
    let RowComponent=render(<Row row={{quizName:"waterQuiz",customerName:"mike"}} onPressHandler={()=>{}}/>).toJSON();
    expect(RowComponent).toMatchSnapshot();
 })