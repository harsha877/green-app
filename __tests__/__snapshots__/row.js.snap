// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row component snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#003f69",
        "borderColor": "black",
        "borderRadius": 10,
        "height": 65,
        "margin": 10,
        "paddingHorizontal": 10,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 16,
          "textAlign": "center",
        }
      }
    >
      "waterQuiz" By Mike
    </Text>
  </View>
</View>
`;
